ros_bridge_interface/src/ros_bridge_interface.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import rospy
+from std_msgs.msg import String
+import json
+import time
+
+
+class RosBridgeInterface:
+    def __init__(self):
+        rospy.init_node('ros_bridge_interface_server', anonymous=True)
+        self.interface_action_sub = rospy.Subscriber("ui/robot/call_action", String, self.interface_action_callback)
+        self.set_screen_pub = rospy.Publisher("ui/app/set_screen", String, queue_size=10)
+
+    def interface_action_callback(self, msg):
```

Comment:
I see this is the callback for the interface to react to the data sent from the tablet. Where does this send the data back to the mission planner? Will there be a service called on the interface node which kicks off the publisher to send the status to the tablet and then the tablet responses with user input and then input is then sent back as a response for the service?

Reply:
The links between rb interface and mission planner need to be implemented. Services, actions, and topics that communicate with mission planner should be defined here, and the data recieved in this callback should be processed and forwarded to the correct service/action/topic